2024-10-06 14:31:53,423 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-06 14:31:58,093 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-06 14:32:05,378 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-06 14:32:06,257 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-06 14:32:47,917 ERROR: Unhandled exception: inventario/reporte.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:316]
2024-10-06 21:39:49,679 ERROR: Error inesperado al crear item: (psycopg2.errors.InvalidTextRepresentation) la sintaxis de entrada no es válida para tipo integer: «»
LINE 3: ...ms_inventario.codigo = '0001' AND items_inventario.id != '' 
                                                                    ^

[SQL: SELECT items_inventario.id AS items_inventario_id, items_inventario.codigo AS items_inventario_codigo, items_inventario.nombre AS items_inventario_nombre, items_inventario.descripcion AS items_inventario_descripcion, items_inventario.tipo AS items_inventario_tipo, items_inventario.categoria AS items_inventario_categoria, items_inventario.precio AS items_inventario_precio, items_inventario.stock AS items_inventario_stock, items_inventario.fecha_creacion AS items_inventario_fecha_creacion, items_inventario.fecha_actualizacion AS items_inventario_fecha_actualizacion 
FROM items_inventario 
WHERE items_inventario.codigo = %(codigo_1)s AND items_inventario.id != %(id_1)s 
 LIMIT %(param_1)s]
[parameters: {'codigo_1': '0001', 'id_1': '', 'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/9h9h) [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\inventario\inventario_routes.py:48]
2024-10-06 22:00:14,923 ERROR: Unhandled exception: inventario/reporte.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:316]
2024-10-06 22:02:45,985 ERROR: Unhandled exception: inventario/reporte.html [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:316]
2024-10-06 22:23:04,332 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-06 22:26:05,375 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-06 22:31:13,320 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-06 22:31:35,722 ERROR: Error al obtener categorías: name 'distinct' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\inventario\inventario_routes.py:228]
2024-10-06 22:35:55,537 ERROR: Error al obtener categorías: name 'distinct' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\inventario\inventario_routes.py:228]
2024-10-06 23:18:06,857 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-06 23:18:59,995 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:261]
2024-10-07 10:31:42,534 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:31:47,192 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:31:51,805 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:35:02,090 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:36:16,425 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:38:42,751 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:38:43,781 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:39:01,248 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:39:05,158 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:39:23,197 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:39:28,352 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:39:39,332 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:39:45,583 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:39:46,608 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:40:00,304 ERROR: Unhandled exception: Multiple classes found for path "Usuario" in the registry of this declarative base. Please use a fully module-qualified path. [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 10:56:50,769 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:56:51,600 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:56:59,341 ERROR: Unhandled exception: Multiple classes found for path "Usuario" in the registry of this declarative base. Please use a fully module-qualified path. [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 10:59:50,203 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 10:59:56,804 ERROR: Unhandled exception: Multiple classes found for path "Empresa" in the registry of this declarative base. Please use a fully module-qualified path. [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 11:02:11,042 INFO: CalculAI startup [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:264]
2024-10-07 11:03:09,557 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 11:03:13,831 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 11:41:41,339 ERROR: Error al obtener categorías: name 'distinct' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\inventario\inventario_routes.py:228]
2024-10-07 11:43:17,403 ERROR: Error al obtener categorías: name 'distinct' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\inventario\inventario_routes.py:228]
2024-10-07 11:48:18,484 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 11:49:15,867 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:02:29,677 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:04:22,514 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:07:56,518 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:08:01,273 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:08:01,276 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:09:36,798 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:10:13,371 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:11:09,966 ERROR: Unhandled exception: 'user_id' is an invalid keyword argument for Campaign [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:11:09,972 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:11:17,367 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:14:51,317 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:15:07,526 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:15:10,643 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:33:49,318 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:45:41,723 ERROR: Error al obtener categorías: name 'distinct' is not defined [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\inventario\inventario_routes.py:228]
2024-10-07 12:46:24,663 ERROR: Unhandled exception: Entity namespace for "contact" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
2024-10-07 12:46:29,246 ERROR: Unhandled exception: Entity namespace for "campaign" has no property "user_id" [in C:\Users\el_re\OneDrive - Sendiu\Desktop\CalculAI\ERP\app.py:323]
